index.js

```diff
@@ -310,6 +314,36 @@ module.exports = {
       return pull(ssb.box2.listGroupIds({ live: !!opts.live }), paraMap(get, 4))
     }
 
+    function getRootIdOfMsgAuthor(groupRootMsgId, cb) {
+      ssb.db.get(groupRootMsgId, (err, rootMsg) => {
+        // prettier-ignore
+        if (err) return cb(clarify(err, "couldn't get root msg for finding root feed"))
+
+        ssb.metafeeds.advanced.findById(rootMsg.author, (err, groupFeed) => {
+          // prettier-ignore
+          if (err) return cb(clarify(err, "couldn't get root msg for finding root feed"))
+
+          ssb.metafeeds.advanced.findById(
+            groupFeed.parent,
+            (err, shardFeed) => {
+              // prettier-ignore
+              if (err) return cb(clarify(err, "couldn't find parent of group feed"))
+
+              ssb.metafeeds.advanced.findById(
```

Comment:
@Powersource I think we want a method on meta feeds - `findRootFeedId(subfeedId,cb)

Reply:
@mixmix made a PR https://github.com/ssbc/ssb-meta-feeds/pull/119